pipelines/utils/crawler_camara_dados_abertos/flows.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# register flow in prefect
+
+from datetime import timedelta
+
+from prefect import Parameter, case
+from prefect.run_configs import KubernetesRun
+from prefect.storage import GCS
+from prefect.tasks.prefect import create_flow_run, wait_for_flow_run
+from pipelines.constants import constants
+from pipelines.utils.crawler_camara_dados_abertos.tasks import (
+    save_data,
+    check_if_url_is_valid,
+)
+from pipelines.utils.crawler_camara_dados_abertos.constants import update_metadata_variable_dictionary
+from pipelines.utils.constants import constants as utils_constants
+from pipelines.utils.decorators import Flow
+from pipelines.utils.execute_dbt_model.constants import constants as dump_db_constants
+from pipelines.utils.metadata.tasks import update_django_metadata
+from pipelines.utils.tasks import (
+    create_table_and_upload_to_gcs,
+    log_task,
+    get_current_flow_labels,
+    rename_current_flow_run_dataset_table,
+)
+
+# ------------------------------ TABLES UNIVERSAL -------------------------------------
+
+with Flow(
+    name="BD template - Camara Dados Abertos", code_owners=["trick"]
+) as flow_camara_dados_abertos:
+    # Parameters
+    dataset_id = Parameter(
+        "dataset_id", default="br_camara_dados_abertos", required=True
+    )
+    table_id = Parameter(
+        "table_id",
+        required=True,
+    )
+    materialization_mode = Parameter(
+        "materialization_mode", default="prod", required=False
+    )
+    materialize_after_dump = Parameter(
+        "materialize_after_dump", default=True, required=False
+    )
+    dbt_alias = Parameter("dbt_alias", default=True, required=False)
+
+    rename_flow_run = rename_current_flow_run_dataset_table(
+        prefix="Dump: ",
+        dataset_id=dataset_id,
+        table_id=table_id,
+        wait=table_id,
+    )
+
+    update_metadata = Parameter("update_metadata", default=True, required=False)
+
+    with case(check_if_url_is_valid(table_id), True):
```

Comment:
nesse caso se a url não for válida ele não vai levantar nenhum erro né? Isso me parece ruim, pq não vamos ser notificados caso acontece alguma mudança com a url